packages/selectInput/components/SelectInput.tsx

```diff
@@ -99,6 +99,7 @@ class SelectInput extends React.PureComponent<
       showInputLabel,
       inputLabel,
       tooltipContent,
+      hintContent,
```

Comment:
💡  In line #116 we could replace 

```
        hintContent={this.props.hintContent}
```

with

```
        hintContent={thintContent}
```

because of this now.